README.md

```diff
@@ -56,6 +56,7 @@ A list of projects hosted on Railway.
 
 - [Long-term visas](https://longtermvisas.com/): A platform built to help people relocate or immigrate to a new country by empowering them with all the information and support they would need.
 - [Gradient Avatars](https://avatars.jakerunzer.com/): Generate gradient profile images based on some text
+- [Discord Welcome-Bot](https://welcome-bot.github.io/): A multi-purpose discord bot that helps you monitor (welcome and goodbye logs), moderate (kick or ban users who spam your server), etc...
```

Comment:
```suggestion
- [Discord Welcome-Bot](https://welcome-bot.github.io/): A multi-purpose discord bot that helps you monitor (welcome and goodbye logs), moderate (kick or ban users who spam your server), etc. ([Source code](https://github.com/Welcome-Bot/welcome-bot))
```

I noticed the bot is open-source. Maybe also add a link to the source code?

Reply:
Yeah, you are right